.github/workflows/update-flathub.yml

```diff
@@ -40,4 +42,4 @@ jobs:
           git config --global user.name "GitHub Workflow"
           git config --global user.email "cockpituous@cockpit-project.org"
           src/containers/flatpak/update-flathub "$(realpath *.tar.xz)" "$(realpath flathub)"
-          git -C flathub push origin HEAD
+          git -C flathub push git@github.com:cockpit-project/org.cockpit_project.CockpitClient HEAD
```

Comment:
Feels a little magic that this pushes the "2XX" branch name, but I suppose that's git for you :grin: It obviously worked.